tools/export_program_info/CMakeLists.txt

```diff
@@ -4,26 +4,28 @@
 add_executable("export_program_info"
   export_program_info.cpp
   main.cpp
-  ${CMAKE_SOURCE_DIR}/libs/api_common/registry_helper.cpp
   ${CMAKE_SOURCE_DIR}/libs/api_common/store_helper_internal.cpp
   ${CMAKE_SOURCE_DIR}/libs/api_common/utilities.cpp
 )
 
 target_include_directories("export_program_info" PRIVATE
+  "${CMAKE_SOURCE_DIR}/external/usersim/inc"
   "${CMAKE_SOURCE_DIR}/include"
   "${CMAKE_SOURCE_DIR}/include/user"
-  "${CMAKE_SOURCE_DIR}/libs/platform"
-  "${CMAKE_SOURCE_DIR}/libs/platform/user"
   "${CMAKE_SOURCE_DIR}/libs/api"
   "${CMAKE_SOURCE_DIR}/libs/execution_context"
+  "${CMAKE_SOURCE_DIR}/libs/platform"
+  "${CMAKE_SOURCE_DIR}/libs/platform/user"
+  "${CMAKE_SOURCE_DIR}/libs/store_helper"
+  "${CMAKE_SOURCE_DIR}/libs/store_helper/user"
```

Comment:
Why do we need to add `libs/store_helper/user` to the include path?

Reply:
This can be safely removed, including `/libs/store_helper`.